### ISSUE TITLE
Multi-resource CRUD operation actions

### ISSUE THREAD

**Issue by jmeas** [jmeas/redux-resource#138]
Right now, operations across resources require one distinct updates to the store (one per resource) since the action creators only affect one slice. There should be a bulk actions extension that would let you do bulk ops across many resources. Just one idea, but something like:

```js
{
  type: 'MULTI_READ_RESOURCES_PENDING',
  resources: {
    books: [ ... ],
    people: [ ... ]
  }
}
```

This would be very useful for the JSON API extension (#38), where you can do things like recieve back compound documents (and also for non-JSON API apis that just throw together a bunch of arbitrary data from all over the place in responses 😛 )

**Comment by jmeas** [comment]
I'm going to merge this into the JSON API extension (#38)

**Comment by jmeas** [comment]
This will prob be separate from the JSON API lib actually, since JSON API still operates on one resource at a time. The difference is this:

```js
// JSON API
{
  type: 'READ_RESOURCES_SUCCEEDED',
  resourceName: 'books',
  resources: [{ ... }, { ... }],
  // other stuff in here
  included: [{ ... }, { ... }]
}
```

```js
// Multi resources
{
  type: 'MULTI_READ_RESOURCES_SUCCEEDED',
  resources: {
    books: [{ ... }, { ... }],
    people: [{ ... }, { ... }]
  }
}
```

For the most part, JSON API has a format I'd recommend using on the client even when your backend doesn't send messages in the JSON API format. This is one the exception, though, because the `included` array is awkward ([the JSON API authors now agree it should have been an object key'd off by type](https://github.com/json-api/json-api/issues/1170#issuecomment-316005935)).

Maybe one day we'll see an updated JSON API, but for now I'll keep these separate.

**Comment by jmeas** [comment]
Reopening as I’m considering this for #297

**Comment by jmeas** [comment]
People prob be like, "wat's up with this guy with his opening and closing of issues."

Well, I'm closing this one again. I have an idea for this, but it's a ways off.

tl;dr,

```
type: REQUEST_PENDING,
crudAction: 'read'
```

^ this separates the action type from the _thing_ that is happening, which is the request. This naturally paves the way for bulk operation support.
